README.md

```diff
@@ -33,6 +33,7 @@ This repository provides technical guidance for users and developers using [Amaz
 * [Assembly Optimization Guide for Graviton Arm64 Processors](arm64-assembly-optimization.md)
 * [Additional resources](#additional-resources)
 * [How To Resources](howtoresources.md)
+* [Blogs](#Blogs)
```

Comment:
My only suggestion is to change this and the heading below to "Blog Posts".

Reply:
thanks, updated.